packages/client/verifiable-credentials/src/presentation/getNfts.ts

```diff
@@ -56,7 +69,7 @@ export function filterVCCompErc721(nfts: CrossmintWalletNft[]): Nft[] {
 }
 
 export async function getWalletVcCompatibleNfts(chain: string, wallet: string): Promise<Nft[]> {
-    const nfts = await getWalletNfts(chain, wallet);
+    const nfts = await getWalletVCNfts(chain, wallet);
     if (nfts == null) {
         throw new Error("Failed to get nfts");
     }
```

Comment:
Do we need the filterVCCompErc721 anymore? Isn't this ensured by the credentials-specific endpoint now?

Reply:
Yes but that method also projects them to the correct type, the type returned by the wallet api is different